docs/data.ts

```diff
@@ -48,6 +48,7 @@ export const webcontainersLinks = [
   { text: 'Browser configuration', link: '/platform/webcontainers/browser-config' },
   { text: 'Project configuration', link: '/platform/webcontainers/project-config' },
   { text: 'Turbo package manager', link: '/platform/webcontainers/turbo-package-manager' },
+  { text: 'Troubleshooting', link: '/platform/webcontainers/troubleshooting' },
```

Comment:
Instead of "troubleshooting", I think it'd make more sense to call this "Runtime limitations" or something like that, where we can explain what WebContainers can and cannot do.

For instance, I was just triaging an issue in webcontainer-core about how we can't register service workers in the pages served by localservice.

Reply:
I think this is a good idea but a separate page (or a subpage) - we can link to these sections from there but my assumption is that not that every known limitation will result in a bug. What do you think?

The troubleshooting page offers "problems - solutions" type of content and I think it might be really helpful to have one place where you know that all your troubles will be addressed. What do you think?

Reply:
I honestly don't have a hard opinion here. Troubleshooting is more general whereas "Runtime limitations" is more specific, and I think limitations can be seen as troubleshooting. So whatever works for me.

Reply:
"Troubleshooting" for me sounds like "FAQ": a bag of varied things for folks that are lost. I'd rather improve the core doc pages so people can search appropriately, and maybe have an FAQ/troubleshooting page that is just a listicle linking to other pages/sections.

:shrug:

Reply:
Ok, what structure do you suggest? Is it `Troubleshooting -> Runtime limitations` or what exactly do you have in mind?